### PR TITLE
Update docs for latest fastlane release (actions.md, available-plugin…

### PR DESCRIPTION
**Pull request by joshdholtz**
…s.md) 🚀

Changes from https://github.com/fastlane/fastlane/pull/11991

### CODE REVIEW

File: docs/actions.md [hunk 1]
```diff
@@ -77,15 +77,15 @@ Action | Description
 <a href="/actions/verify_xcode">verify_xcode</a> | Verifies that the Xcode installation is properly signed by Apple
 <a href="/actions/xcode_install">xcode_install</a> | Make sure a certain version of Xcode is installed
 <a href="/actions/verify_pod_keys">verify_pod_keys</a> | Verifies all keys referenced from the Podfile are non-empty
-<a href="/actions/xcarchive">xcarchive</a> | Archives the project using `xcodebuild`
-<a href="/actions/xcclean">xcclean</a> | Cleans the project using `xcodebuild`
-<a href="/actions/spm">spm</a> | Runs Swift Package Manager on your project
 <a href="/actions/build_app">build_app</a> | Alias for the `build_ios_app` action
+<a href="/actions/xcbuild">xcbuild</a> | Builds the project using `xcodebuild`
```
Comment (janpio) [inline docs/actions.md]: This is the noise I meant ;)

File: docs/actions.md [hunk 2]
```diff
@@ -313,20 +313,20 @@ Action | Description
 <a href="/actions/build_and_upload_to_appetize">build_and_upload_to_appetize</a> | Generate and upload an ipa file to appetize.io
 <a href="/actions/update_icloud_container_identifiers">update_icloud_container_identifiers</a> | This action changes the iCloud container identifiers in the entitlements file
 <a href="/actions/sh">sh</a> | Runs a shell command
-<a href="/actions/min_fastlane_version">min_fastlane_version</a> | Verifies the minimum fastlane version required
+<a href="/actions/setup_travis">setup_travis</a> | Setup the keychain and match to work with Travis CI
```
Comment (janpio) [inline docs/actions.md]: even worse noise

File: docs/actions.md (hunk 1 above)
Comment (joshdholtz) [reply docs/actions.md]: Ha, I see what you mean now 😱

File: docs/actions.md (hunk 2 above)
Comment (revolter) [reply docs/actions.md]: Are these randomized every time?

Comment (janpio) [reply docs/actions.md]: yep, each docs PR has a slightly different order of things

**Pull request merged**